global-variables.ps1

```diff
@@ -87,6 +87,18 @@ $METRICS_DCOS_WINDOWS_GIT_REPO_DIR = Join-Path $METRICS_DIR "dcos-windows"
 $METRICS_MESOS_JENKINS_GIT_REPO_DIR = Join-Path $METRICS_DIR "mesos-jenkins"
 $METRICS_BUILD_BASE_URL = "$LOG_SERVER_BASE_URL/metrics-build"
 
+# AdminRouter configurations
+$ADMINROUTER_SERVICE_NAME = "dcos-adminrouter"
+$ADMINROUTER_AGENT_PORT = 9003
+$ADMINROUTER_DIR = Join-Path $DCOS_DIR "adminrouter"
+$ADMINROUTER_GIT_REPO_DIR = Join-Path $ADMINROUTER_DIR "src\github.com\dcos\dcos-adminrouter"
+$ADMINROUTER_BUILD_OUT_DIR = Join-Path $ADMINROUTER_DIR "build-output"
+$ADMINROUTER_BUILD_LOGS_DIR = Join-Path $ADMINROUTER_BUILD_OUT_DIR "logs"
+$ADMINROUTER_BUILD_BINARIES_DIR = Join-Path $ADMINROUTER_BUILD_OUT_DIR "binaries"
+$ADMINROUTER_MESOS_JENKINS_GIT_REPO_DIR = Join-Path $ADMINROUTER_DIR "mesos-jenkins"
+$ADMINROUTER_BUILD_BASE_URL = "$LOG_SERVER_BASE_URL/adminrouter-build"
+$ADMINROUTER_APACHE_DIR = "Apache24"
+
```

Comment:
Add `$REMOTE_ADMINROUTER_BUILD_DIR` after the other ones above. The line:

```
$REMOTE_ADMINROUTER_BUILD_DIR = "/data/adminrouter-build"
```

should inserted [here](https://github.com/Microsoft/mesos-jenkins/blob/4885004304e635ff9c39d1022851a48cd03b3efd/global-variables.ps1#L10).